packages/dds/sequence/src/IntervalCollectionValues.ts

```diff
@@ -9,101 +9,19 @@ import {
 	IFluidSerializer,
 } from "@fluidframework/shared-object-base/internal";
 
-import type { IntervalCollection } from "./intervalCollection.js";
-import {
-	IIntervalCollectionOperation,
-	IIntervalCollectionType,
-	ISerializableIntervalCollection,
-	ISerializedIntervalCollection,
-} from "./intervalCollectionMapInterfaces.js";
-import { IntervalOpType } from "./intervals/index.js";
-
-/**
- * A local value to be stored in a container type DDS.
- */
-export interface ILocalIntervalCollection {
-	/**
-	 * Type indicator of the value stored within.
-	 */
-	readonly type: string;
-
-	/**
-	 * The in-memory value stored within.
-	 */
-	readonly value: IntervalCollection;
-
-	/**
-	 * Retrieve the serialized form of the value stored within.
-	 * @param serializer - Data store runtime's serializer
-	 * @param bind - Container type's handle
-	 * @returns The serialized form of the contained value
-	 */
-	makeSerialized(
-		serializer: IFluidSerializer,
-		bind: IFluidHandle,
-	): ISerializedIntervalCollection;
-}
+import { type IntervalCollection } from "./intervalCollection.js";
+import { ISerializableIntervalCollection } from "./intervalCollectionMapInterfaces.js";
 
 export function makeSerializable(
```

Comment:
this PR is probably best reviewed split, there are lots of type removals, and collapsing of implementations to the minimal necessary set